maestro-test/src/test/resources/051_text_random.yaml

```diff
@@ -0,0 +1,13 @@
+appId: com.example.app
+---
+
+- inputTextRandom:
```

Comment:
- How about `inputRandomText` instead of `inputTextRandom` to make it read more like an English sentence?
- For the same reason, let's add aliases for Yaml commands: `inputRandomText`, `inputRandomNumber`, `inputRandomEmail`, `inputRandomPersonName`. They could all use the same `InputTextRandomCommand` under the hood